kas/include/rockchip.yml

```diff
@@ -0,0 +1,25 @@
+header:
+  version: 14
+
+repos:
+  meta-arm:
```

Comment:
Is there a special requirement concerning the revision of `meta-arm`? If not, please remove and use `include/arm.yml`.

Reply:
Ok I will test that.

Reply:
Ok I checked it is a requirement of meta-rockchip  :-)
https://git.yoctoproject.org/meta-rockchip/tree/conf/layer.conf